src/queries/mediaContent.js

```diff
@@ -1,14 +1,21 @@
 import * as SecureStore from 'expo-secure-store';
 
-import { namespace, secrets } from '../config';
+import { consts, namespace, secrets } from '../config';
 
-export const uploadMediaContent = async (image, contentType) => {
+const { MEDIA_TYPES } = consts;
+
+export const uploadMediaContent = async (
+  content,
+  contentType,
+  contentName = 'image',
+  type = 'jpg'
+) => {
   const formData = new FormData();
   formData.append('media_content[content_type]', contentType);
   formData.append('media_content[attachment]', {
-    uri: image.uri,
-    type: 'image/jpg',
-    name: 'image.jpg'
+    uri: content.uri || content.cachedAttachment,
+    type: type === MEDIA_TYPES.IMAGE ? 'image/jpg' : 'application/pdf',
```

Comment:
can we turn this check around to have explicitly checked for pdf and fallback to image otherwise as we had before?

Reply:
i cannot see a code change to check PDF and set image otherwise?

Reply:
ok, now it's there!

![image](https://github.com/user-attachments/assets/f669d862-7158-4809-9aaf-692dc10acf19)